.github/workflows/kustomize-deploy-commit.yml

```
@@ -3,18 +3,22 @@ on:
   workflow_call:
     inputs:
       KUBERNETES_REPO:
+        description: 'The GitHub repository to deploy to'
         required: true
         type: string
-      GITHUB_EMAIL:
+      KUBERNETES_REPO_REF:
+        description: 'The GitHub repository reference to deploy to'
         required: true
         type: string
-      APPLICATION:
+        default: 'master'
+      GITHUB_EMAIL:
         required: true
         type: string
-      CLUSTER_A:
+      APPLICATION:
         required: true
         type: string
```

```suggestion
      APPLICATION:
        required: true
        type: string
    KUSTOMIZE_PATH:
        type: string
        default: kustomize/overlays/instances
```
This path is different for legacy-smc